commec/config/io_parameters.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021-2024 International Biosecurity and Biosafety Initiative for Science
+"""
+Defines the `Input and Output Screen Parameters` class, and associated dataclasses.
+"""
+import os
+import glob
+import argparse
+from dataclasses import dataclass
+import subprocess
+
+@dataclass
+class ScreenInputParameters:
```

Comment:
Sure, as a member variable of Screen, it is labelled as inputs, so having Inputs : ScreenConfig = la de da, makes sense to me. I do like the word inputs, as it clearly labels these as input configurations from the user - rather than derived configurations that are hard coded or calculated from other things.